src/bar-chart/series-vertical.component.ts

```diff
@@ -203,9 +204,11 @@ export class SeriesVerticalComponent implements OnChanges {
       }
 
       let tooltipLabel = formattedLabel;
+      bar.ariaLabel = formattedLabel + ' ' + value.toLocaleString();
```

Comment:
The accessibility issue I mentioned in a separate comment on this PR could be fixed potentially by making the arialabel structured like this instead:

`'Label: ' + formattedLabel + ', data: ' + value.toLocaleString();`